README.md

```diff
@@ -14,7 +14,7 @@ Our program needs to have a way to bundle up and operate on all the information
 
 How can we tell our Ruby program to deal with these dogs? Well, we can write a `Dog` class that produces individual dog objects, each of which contains all the information and behaviors of an individual dog.
 
-Think of a class like a factory that produces products on an assembly line. The factory produces a fleet of similar products, all based off of the same blueprint. A Ruby class is both the factory and the blueprint - it contains the instructions for creating new objects and it has the ability to create those objects.
+Think of a class like the blueprint that defines how to build an object.  The `Dog` class is different from an individual dog just as the blueprints that show how to build a house are not the actual house. A Ruby class both contains the instrucions for creating new objects and has the ability to create those objects. It acts like an assembly line which produces a series of individual dog objects all based off the same template. 
```

Comment:
"instrucions" -> "instructions"

I like this analogy -- maybe we could clarify the assembly line bit by changing the last sentence slightly, e.g.:

"Calling `.new` on a class is like getting an individual dog object from an assembly line that produces a series of objects based on the `Dog` template."

I'm sure we can wordsmith that a bit, but the main thing to emphasize is that the `.new` method is essential to following our blueprint.